CDCgov/cdc-open-viz
# Fix: [DEV-9001] date format for table

## [Replace With Ticket Number]



## Testing Steps
Open Bar/Line/Combo charts.
Switch to Date time or Date scale
Enter date format for text field "Data Table date display format" which updates only data table if entered.
The data table date format should be changed entered date format.



## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
